cpg_pipes/pipeline/stages.py

```diff
@@ -0,0 +1,121 @@
+"""
```

Comment:
I think this file should be called `stage_targets.py`, I got quite confused about the naming while reviewing this PR.

Reply:
Renamed to `stage_subclasses.py`